aceds/active_medicaid_case/request.xml

```diff
@@ -0,0 +1,29 @@
+<!-- we can just send that ClientInquiryRequest payload and only populate last name, first name, DOB, and SSN -->
+<tns:ClientInquiryRequest xmlns:tns="http://xmlns.dhs.dc.gov/DCAS/ESB/ACDService/V1">
+<tns:RequestNumber></tns:RequestNumber>
+<tns:ACEDS_CONDITION-CODE/>
+<tns:ACEDS_CONDITION-DESC/>
+<tns:ACEDS_USER-ID></tns:ACEDS_USER-ID>
+<tns:ACEDS_CLIENT-NUMBER-I></tns:ACEDS_CLIENT-NUMBER-I>
+<tns:ACEDS_LAST-NAME-I>Garcia</tns:ACEDS_LAST-NAME-I>
+<tns:ACEDS_FIRST-NAME-I>Mikey</tns:ACEDS_FIRST-NAME-I>
+<tns:ACEDS_MIDDLE-NAME-I/>
+<tns:ACEDS_SUFFIX-I xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:nil="1"/>
+<tns:ACEDS_DATE-OF-BIRTH-I>6061969</tns:ACEDS_DATE-OF-BIRTH-I>
+<tns:ACEDS_SEX-I>M</tns:ACEDS_SEX-I>
```

Comment:
We'll remove gender, and just have the identifying PII set be {firstname, lastname, SSN, DOB}. Particularly since the documentation above suggests that we're not sending gender as part of our signup call from the client anyway.

However, don't change it for this PR. I'll approve the PR and then ask Patrick to update this.